### PR TITLE
Final touchups

### PR DESCRIPTION
**Pull request by NoahHakansson**
Adds `uninstall.sh` and improves `install.sh` to use curl instead of `git clone`
Faster because curl only gets the most up-to-date source code, and no `.git` stuff.

So pushing many binaries will not make downloads slower now.

Changes tool output to be more readable, inspired by `eslint` output.

Fix `-h` and `--help` flag instead of `--h`. Now also doesn't crash if given no args and instead prints help menu.

Some typo fixes and small improvements to `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
 ### Doesn't steal any data! :D
-The FastText model is pre-trained and of course, will not by any means save any data of the projects it's used in. The model is downloaded from our gitlab repo and ready to be used afterwards.
+The FastText model is pre-trained and of course, will not by any means save any data of the projects it's used in. The model is downloaded from a GitLab repo and ready to be used afterwards.
 
 ## Goals of this project
-Gosplat's and our goals are to create a nice, quick and easy way to analyze possible holes in your or your team's file structure. It may give helpful hints which can aid in creating a more maintainable enviorment for future projects to come!
+Gosplat's and our goals are to create a nice, quick and easy way to analyze possible holes in your or your team's file structure. It may give helpful hints which can aid in creating a more maintainable environment for future projects to come!
```
Comment (Gabriel-Ivarsson) [inline README.md:12]: Not your fault, mine but it's "you or your team's file structure", changing "your" to "you"

File: install.sh [hunk 2]
```diff
@@ -3,12 +3,47 @@
...
+echo "Copying built binary to $binary_dest"; echo
+\cp gosplat "$binary_dest"
+
+# clean up
+rm -f $tempfile
+rm -rf $repo_dir
+rm -f "gosplat"
+
+# done
+echo "Installation done!"
```
Comment (Gabriel-Ivarsson) [inline install.sh:49]: It's not done until the python dependencies also are pulled to the computer, but i can fix that part

File: main.go [hunk 3]
```diff
@@ -11,13 +11,17 @@ import (
 
 func printHelp() {
 	fmt.Printf("Gosplat v1\n" +
-		" gosplat --h , prints commands\n" +
+		" gosplat -h/--help , prints this help text\n" +
```
Comment (Gabriel-Ivarsson) [inline main.go:14]: was fixed in another PR

File: main.go [hunk 4]
```diff
@@ -11,13 +11,17 @@ import (
...
 		" gosplat [input dir/file] , runs directory against model\n")
 }
 
 func checkForFlags(args []string) uint8 {
+	if len(args) == 1 {
```
Comment (Gabriel-Ivarsson) [inline main.go:19]: Very nice

File: src/python_helper/compare_helper.py [hunk 5]
```diff
@@ -88,8 +88,8 @@ def check_function(self, function_name: str, old_package_name: str):
             function_name)
         if best_match_package != old_package_name:
             if self.mitigate_FPs(function_name, old_package_name, 1) == False:
-                print(
-                    red(f"Function: '{function_name}' in '{old_package_name}' package, may NOT be in the best matching package, consider placing it in '{best_match_package}' or choosing a better name for {old_package_name}!"))
+                print(red("\nerror: "),
```
Comment (Gabriel-Ivarsson) [inline src/python_helper/compare_helper.py:91]: Much cleaner

**Review by Gabriel-Ivarsson — APPROVED** [review]
Merge when some of the comments are resolved, others are just to click resolved right away. Good work

File: install.sh (hunk 2 above)
Comment (NoahHakansson) [reply install.sh:49]: Not sure if we want auto install pip packages, better to let the user have that control over their computer. And the error is very obvious and clearly states that X package need to be installed with pip.

File: main.go (hunk 3 above)
Comment (NoahHakansson) [reply main.go:14]: Don't think it was, i looked for it firstly in `dev`

**Pull request merged**